### PR TITLE
Active effects

### LINKED ISSUE #470: Opening screen tells the story
**Describe the solution you'd like**
Have the "backstory" (#392) be displayed on the opening screen.

### PR DESCRIPTION
**Pull request by groadybones**
Fixes #470, #463, #310

#### Overview of changes

-Added a backstory to the sideNav panel
-Made it so there is a full set of safety cards for every 2 players in the game
-Minor deck balancing

Reviewer: @johnanvik

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (johnanvik) [inline .vscode/launch.json]: Do we need this comment? Looks like it was generated. Check throughout the files for this.

File: src/classes/Models/Deck.js [hunk 2]
```diff
@@ -100,18 +100,18 @@ export default class Deck {
     for (let k = 0; k < value; k++) {
       for (let i = 0; i < cardDeck.length; i++) {
         for (let j = 0; j < cardDeck[i].howMany; j++) {
-          if (cardDeck[i].type === 'FIREWALL' && !fwPlayed) {
+          if (cardDeck[i].type === 'FIREWALL' && !fwPlayed && (k === 1 || k === 3)) {
```
Comment (johnanvik) [inline src/classes/Models/Deck.js]: What is this check for?

File: src/classes/Models/Deck.js [hunk 3]
```diff
@@ -100,18 +100,18 @@ export default class Deck {
...
             this.cards.push(new Card(cardId, cardDeck[i].cardValue, cardDeck[i].type, cardDeck[i].imgSrc))
             cardId++
-            fwPlayed = true
-          } else if (cardDeck[i].type === 'GENERATOR' && !genPlayed) {
+            // fwPlayed = true
```
Comment (johnanvik) [inline src/classes/Models/Deck.js]: Dead code. Remove. Look through other changes for dead code.

File: src/components/SharedComponents/SettingsComponent.vue [hunk 4]
```diff
@@ -99,7 +106,8 @@
         aiOpponents: ['Flash', 'Joker', 'Aquaman', 'Superman'],
         typesOfGames: ['Short (100)', 'Medium (150)', 'Long (200)'],
         isTutorial: false,
-        tutorialBegin: false
+        tutorialBegin: false// ,
+        // showBackstory: true
```
Comment (johnanvik) [inline src/components/SharedComponents/SettingsComponent.vue]: Dead code

File: src/components/SharedComponents/Stack.vue [hunk 5]
```diff
@@ -131,6 +131,13 @@ export default {
 
         return (thisActiveCard !== undefined && thisActiveCard.type === 'G' && !this.getCurrentPlayer().hasPowerOutage)
       },
+      /* groupableStack () {
```
Comment (johnanvik) [inline src/components/SharedComponents/Stack.vue]: Dead code

File: src/components/SharedComponents/Stack.vue [hunk 6]
```diff
@@ -294,6 +301,18 @@ export default {
       cardAddClicked () {
         this.$emit('cardAddClicked', this.id)
       },
+      /**
```
Comment (johnanvik) [inline src/components/SharedComponents/Stack.vue]: Remove dead code.

File: .vscode/launch.json (hunk 1 above)
Comment (groadybones) [reply .vscode/launch.json]: We talked about each of these in our meeting so I will just resolve them all, this comment is to make note that they've all been addressed

**Pull request merged**